File: MovieTutorial/MovieTutorial.Web/Modules/MovieTutorial/Employee/EmployeeForm.cs

namespace MovieTutorial.Forms
{
    using Serenity;
    using Serenity.ComponentModel;
    using Serenity.Data;
    using System;
    using System.ComponentModel;
    using System.Collections.Generic;
    using System.IO;

    [FormScript("Employee")]
    [BasedOnRow(typeof(Entities.EmployeeRow))]
    public class EmployeeForm
    {
        public String Name { get; set; }
        public Int32 Address { get; set; }
        public String Email { get; set; }
    }
}

File: MovieTutorial/MovieTutorial.Web/Modules/Common/Imports/ServerTypings/EmployeeForm.ts


namespace MovieTutorial {
    export class EmployeeForm extends Serenity.PrefixedContext {
        static formKey = 'Employee';
    }

    export interface EmployeeForm {
        Name: Serenity.StringEditor;
        Address: Serenity.IntegerEditor;
        Email: Serenity.StringEditor;
    }

    [['Name', () => Serenity.StringEditor], ['Address', () => Serenity.IntegerEditor], ['Email', () => Serenity.StringEditor]].forEach(x => Object.defineProperty(EmployeeForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}